include/zephyr/net/net_context.h

```diff
@@ -1004,6 +1005,10 @@ static inline int net_context_create_ipv6_new(struct net_context *context,
 					      const struct in6_addr *src,
 					      const struct in6_addr *dst)
 {
+        ARG_UNUSED(context);
```

Comment:
if you are using vscode, the `editorconfig` extension would apply the right indentation for different file extensions automatically for you